contrib/graphite/blueflood.py

```diff
@@ -56,8 +62,9 @@ def calc_res(start, stop):
 class TenantBluefloodFinder(threading.Thread):
   __fetch_multi__ = 'tenant_blueflood'
   __fetch_events__ = 'tenant_blueflood'
+
   def __init__(self, config=None):
-    print("Blueflood Finder v31")
+    logger.debug("Blueflood Finder v31")
```

Comment:
what is the significance of the version 31? The setup.py says it's version 0.0.1